README.md

```diff
@@ -37,4 +38,9 @@ As the scripts are intended to be run from within the Travis CI environment they
 | `TRAVIS_PULL_REQUEST`  | NO     | PR number                                                                                                                                           | NO             |
 | `TRAVIS_BRANCH`        | NO     | Branch name triggering the build                                                                                                                    | NO             |
 | `TRAVIS_TAG`           | NO     | Name of tag triggering the build                                                                                                                    | NO             |
+There is a utility script [here](./configure-travis-env.sh) to set the non-Travis environment variables from the command line.
+It assumes you have the Travis CLI installed and you are already logged into Travis.
 
+Finally, it is worth noting that it is possible to run the scripts locally by setting the envirionment variables. For example:
+
+ `TRAVIS=true RANCHER_TEMPLATE_NAME=c2s-pharmacy-finder TRAVIS_REPO_SLUG=nhsuk/connecting-to-services TRAVIS_PULL_REQUEST=<PR number of an image which still exists> SPLUNK_HEC_TOKEN=<splunk token> GITHUB_ACCESS_TOKEN=<personal access token> ./rancher_pr_deploy.sh nearby-services-api`
```

Comment:
`TEMPLATE_BRANCH_NAME` is missing from this line and also from the table of env vars above?